docs/reference/edot-sdks/features.yml

```diff
@@ -352,17 +352,17 @@ features:
       status: not-available
       min_version: 
     Java:
-      status: not-available
-      min_version: 
+      status: tech-preview
```

Comment:
Is this file typically maintained by developers? Is this something we can eventually use to automate reference docs from? For example, the table you've added in `central-configuration.md`.